<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U63UI4D" failures="2" tests="8" name="com.qa.orangehrm.tests.LoginPageTest" time="107.991" errors="0" timestamp="2022-08-22T09:16:01 BST" skipped="2">
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="verifyLoginPageTitleTest" time="13.023"/>
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="verifyLoginPageTitleTest" time="11.143"/>
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="verifyForgetPassLink" time="22.619"/>
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="verifyForgetPassLink" time="24.179"/>
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="LoginTest" time="17.233">
    <failure message="expected [Lolita S] but found [Manoj c]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Lolita S] but found [Manoj c]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.orangehrm.tests.LoginPageTest.LoginTest(LoginPageTest.java:91)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="LoginTest" time="19.794">
    <failure message="expected [Lolita S] but found [null]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Lolita S] but found [null]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.orangehrm.tests.LoginPageTest.LoginTest(LoginPageTest.java:91)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="login_InvalidTestCase">
    <skipped/>
  </testcase> <!-- login_InvalidTestCase -->
  <system-out/>
  <testcase classname="com.qa.orangehrm.tests.LoginPageTest" name="login_InvalidTestCase">
    <skipped/>
  </testcase> <!-- login_InvalidTestCase -->
  <system-out/>
</testsuite> <!-- com.qa.orangehrm.tests.LoginPageTest -->
